templates/endpoints_v2/cookbooks/vision_language_model/vision_language_model/README.ipynb

```diff
@@ -0,0 +1,285 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Deploy, configure, and serve LLMs \n",
+    "\n",
+    "**⏱️ Time to complete**: 10 min\n",
+    "\n",
+    "Ready for some adventure into multi-modal models? Vision language model support is now added to RayLLM. The current support includes LlaVA-NeXT models with dynamic image resolution. Meta AI’s Chameleon, with its early fusion architecture, will come soon. \n",
+    "\n",
+    "In this example, we will use RayLLM to serve [llava-hf/llava-v1.6-mistral-7b-hf](https://huggingface.co/llava-hf/llava-v1.6-mistral-7b-hf). We will build a Gradio application that brings art to life with ekphrasis, specially designed for kids. Imagine turning kid drawings into whimsical verses and delightful rhymes. Now let’s buckle up and dive into the fun! 🎨🚀\n",
+    "\n",
+    "**Note**: This guide is hosted within an Anyscale workspace, which provides easy access to compute resources. Check out the `Introduction to Workspaces` template for more details.\n",
+    "\n",
+    "## Step 1 - Run the model locally in the Workspace\n",
```

Comment:
Don't just copy paste stuff from the main README. 

I would like the content of this will be something like: 

- An example display of steps in `generate_config.py`.

- Highlight of the main parts of the generated config and what each parameter does? 

- Service deploy command

- Example of using playground to query it (with screen-shots)

- Example of querying it through code (both url and local images) for local images just put the files in an example folder. So that you can remove `download.sh` etc. 

- Example of building a gradio service app (to show port forwarding + development on workspace in such an application env, here you can say instead of deploying a production service we deploy it locally and host a local gradio app to test different versions)

- You can remove qeury_with_url.py and query_with_base64_str.py files.

Reply:
Thanks.
I am reorganizing the materials, these are my thinking process:
An example display of steps in generate_config.py.
> An example display of steps in generate_config.py.
> Highlight of the main parts of the generated config and what each parameter does?
> Service deploy command
If it's important to showcase these, then let's do that in the upper level doc. I would like this doc to focus on things that may be different because of images. i.e. how to query them easily (so playground suggestion makes sense). 

> Example of using playground to query it (with screen-shots)
This makes sense. I am adding it. But may need to wait for prod deploy? Can we move this item to a follow up PR to make progress now?

The rest I have made changes accordingly.

Reply:
> You can remove qeury_with_url.py and query_with_base64_str.py files.

Why do we want to remove this? I see other guides also keeping the query.py file